util/monoclock.c

```diff
@@ -43,6 +44,12 @@ monoclock_get(struct timeval * tv)
 		goto err0;
 	}
 #endif
+#else	/* POSIXFAIL_CLOCK_GETTIME */
```

Comment:
Repeating the `gettimeofday()` lines isn't ideal, but I think it's more readable than doing fancy `#if something && anotherthing`, especially since these variables are long so it would go well over 80 chars.

Reply:
I'd be inclined to rewrite the code by putting some preprocessor logic at the top of the file which sets `USE_MONOTONIC`, `USE_REALTIME`, and `USE_GETTIMEOFDAY` based on whether `CLOCK_MONOTONIC`, `POSIXFAIL_CLOCK_REALTIME`, and `POSIXFAIL_CLOCK_GETTIME` are defined.  I don't like code duplication, and my hunch is that it would be clearer that way.